### PR TITLE
Added ability to put text labels on the mosaic to indicate the fov.

### PR DESCRIPTION
**Pull request by timblosser**

### CODE REVIEW

File: merlin/analysis/generatemosaic.py [hunk 1]
```diff
@@ -165,6 +167,11 @@ def _prepare_mosaic_slice(self, zIndex, dataChannel, micronExtents,
                 inputImage[:, :cropWidth] = 0
                 inputImage[:, inputImage.shape[0] - cropWidth:] = 0
 
+            if self.parameters['draw_fov_labels']:
```
Comment (seichhorn) [inline merlin/analysis/generatemosaic.py]: It'd be preferable to place the labels based on the relative position as opposed to absolute, in case someone uses a smaller format camera.

File: merlin/data/dataorganization.py [hunk 2]
```diff
@@ -112,9 +112,10 @@ def get_data_channel_index(self, dataChannelName: str) -> int:
             # TODO this should raise a meaningful exception if the data channel
             # is not found
         """
-        return self.data[self.data['channelName'].apply(
-            lambda x: str(x).lower()).str.match(
-            dataChannelName.lower())].index.values.tolist()[0]
+        return self.data[self.data['channelName'] == dataChannelName].index.values.tolist()[0]
```
Comment (emanuega) [inline merlin/data/dataorganization.py]: Why is this changed? It seems it could lose backward compatibility by requiring the names to match case.

Comment (seichhorn) [reply merlin/data/dataorganization.py]: I think it's reasonable to remove the case sensitivity in the update as you want. The reason this was changed was because in instances where you had two names that contained the same string (i.e. 'ABCD' and 'ABCDE'), depending on the order they appeared in in the file, it could be impossible to get it to generate a mosaic for the one you wanted to target since it just took the first element of the match previously.

**Comment by seichhorn** [comment]
@emanuega I changed the dataorganization.get_data_channel_index to use case-insensitive matching, but to still require exact match. If you're ok with this then please merge.
